content/documentation/langiumApproach/content.md

```diff
@@ -0,0 +1,331 @@
++++
+fragment = "content"
+title = "Langium-Based DSL Approach"
+weight = 150
+[sidebar]
+  sticky = true
++++
+
+# Unleashing the Power of Textual DSLs
```

Comment:
I think we should not use H1 here, but rather H3 and below. Also I'd remove this heading entirely, as we already have a heading on this page with the same role (main heading of the page). To me personally, "Unleashing the Power" is also a bit too flashy.

So suggest to remove this H1 and push all headings below to H3